src/central_control/scripts/aabb.py

```diff
@@ -0,0 +1,58 @@
+from math import sqrt
```

Comment:
This file, alongside Range and Point, are in the same folder as central control. For some reason when I put them in a different folder, they couldn't be found, even though I didn't see any issue with the imports I had